CHANGELOG.md

```diff
@@ -5,6 +5,39 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### Added
+
+### Changed
+- Change `LineItem`'s `productType` to `category`([#27](https://github.com/velocidi/velocidi-android-sdk/pull/27)).
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
+### Security
+
+### Infrastructural
+
+
+## [Unreleased]
+### Added
+
+### Changed
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
+### Security
+
+### Infrastructural
+
+
```

Comment:
Can you remove these lines? They're duplicated.

Reply:
When changing the CHANGELOG for the release of 0.3.0, I left the `Unreleased` section below the released, which is why this happened. Fixed on da3e1e816bc4ab4c60262f7b569f900c1a0d7187.